<HTML>
<!--This file created 3/12/07 10:22 AM by Claris Home Page version 3.0-->
<HEAD>
   <TITLE>Gene Control</TITLE>
   <META NAME=GENERATOR CONTENT="Claris Home Page 3.0">
   <X-CLARIS-WINDOW TOP=66 BOTTOM=637 LEFT=8 RIGHT=538>
   <X-CLARIS-TAGVIEW MODE=minimal>
</HEAD>
<BODY BGCOLOR="#FFFFFF">
<H1><FONT SIZE="+1">CONTROL OF GENE EXPRESSION</FONT></H1>

<H3><A NAME="Table of Contents"></A>Table of Contents</H3>

<P><A HREF="#The chromosome of E. coli"><B>The chromosome of <I>E.
coli</I></B></A><B><I> </I></B><FONT SIZE="+1"> </FONT><B> |
</B><FONT SIZE="+1"> </FONT><A HREF="#The operon model"><B>The operon
model</B></A><B> |</B><FONT SIZE="+1"> </FONT><A HREF="#Viruses"><B>Viruses</B></A><B>
| </B><A HREF="#Plasmids"><B>Plasmids</B></A><B>
|</B><FONT SIZE="+1"> </FONT><A HREF="#The eukaryotic chromosome"><B>The
eukaryotic chromosome</B></A></P>

<P><A HREF="#Replication of the eukaryotic c"><B>Replication of the
eukaryotic chromosome</B></A><B> | </B><A HREF="#Regulation of eukaryotic gene e1724"><B>Regulation
of eukaryotic gene expression</B></A></P>

<P><A HREF="#Types of chromatin"><B>Types of chromatin</B></A><B> |
</B><A HREF="#The eukaryotic genome"><B>The eukaryotic
genome</B></A><B> | </B><A HREF="#Transcription and processing of"><B>Transcription
and processing of mRNA</B></A></P>

<P><A HREF="#Antibody-coding genes"><B>Antibody-coding
genes</B></A><B> | </B><A HREF="#Viruses and eukaryotes"><B>Viruses
and eukaryotes</B></A><B> | </B><A HREF="#Eukaryotic transposons"><B>Eukaryotic
transposons</B></A></P>

<P><A HREF="#Genes, viruses and cancer"><B>Genes, viruses and
cancer</B></A><B> | </B><A HREF="#Links"><B>Links</B></A></P>

<P><FONT SIZE="+1">While the period from 1900 to the second world war
has been called the "golden age of genetics", we may be in a new
golden (or platinum) age. Recombinant DNA technology allows us to
manipulate the very DNA of living organisms and to make conscious
changes in that DNA. Prokaryote genetic systems are much easier to
study and better understood than are eukaryote systems.</FONT></P>

<H3><FONT SIZE="+1"><A NAME="The chromosome of E. coli"></A>The
chromosome of <I>E. coli </I>| </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back
to Top</FONT></A></H3>

<P><FONT SIZE="+1">The single chromosome of the common intestinal
bacterium <I>E. coli</I> is circular and contains some 4.7 million
base pairs. It is nearly 1 mm long, but only 2nm wide. The chromosome
replicates in a bidirectional method, producing a figure resembling
the Greek letter theta. The </FONT><A HREF="BioBookglossPQ.html#promoter"><FONT SIZE="+1">promoter</FONT></A><FONT SIZE="+1">
is the part of the DNA to which the </FONT><A HREF="BioBookglossR.html#RNA polymerase"><FONT SIZE="+1">RNA
polymerase</FONT></A><FONT SIZE="+1"> binds before opening the
segment of the DNA to be transcribed. </FONT></P>

<CENTER><FONT SIZE="+1"><IMG SRC="bactchromo.gif" WIDTH=362 HEIGHT=322 ALIGN=bottom></FONT></CENTER>

<P><FONT SIZE="+1">Partial gene map of the operons, such as trp and
lac, on a bacterial chromosome.</FONT> Image from Purves et al.,
<U>Life: The Science of Biology</U>, 4th Edition, by Sinauer
Associates (<A HREF="http://www.sinauer.com">www.sinauer.com</A>) and
WH Freeman (<A HREF="http://www.whfreeman.com">www.whfreeman.com</A>),
used with permission.</P>

<P><FONT SIZE="+1">A segment of the DNA that codes for a specific
polypeptide is known as a structural </FONT><A HREF="BioBookglossG.html#gene"><FONT SIZE="+1">gene</FONT></A><FONT SIZE="+1">.
These often occur together on a bacterial chromosome. The location of
the polypeptides, which may be enzymes involved in a biochemical
pathway, for example, allows for quick, efficient transcription of
the mRNAs. Often leader and trailer sequences, which are not
translated, occur at the beginning and end of the region. <I>E.
coli</I> can synthesize 1700 enzymes. Therefore, this small bacterium
has the genes for 1700 different mRNAs. </FONT></P>

<P><FONT SIZE="+1">Lactose, milk sugar, is split by the enzyme
</FONT><FONT SIZE="+1" FACE="Symbol">b</FONT><FONT SIZE="+1">-galactosidase.
This enzyme is inducible, since it occurs in large quantities only
when lactose, the substrate on which it operates, is present.
Conversely, the enzymes for the amino acid tryptophan are produced
continuously in growing cells unless tryptophan is present. If
tryptophan is present the production of tryptophan-synthesizing
enzymes is repressed. </FONT></P>

<H3><FONT SIZE="+1"><A NAME="The operon model"></A>The Operon Model |
</FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back to
Top</FONT></A></H3>

<P><FONT SIZE="+1">The operon model of prokaryotic gene regulation
was proposed by Fancois Jacob and Jacques Monod. Groups of genes
coding for related proteins are arranged in units known as operons.
An operon consists of an operator, promoter, regulator, and
structural genes. The regulator gene codes for a repressor protein
that binds to the operator, obstructing the promoter (thus,
transcription) of the structural genes. The regulator does not have
to be adjacent to other genes in the operon. If the repressor protein
is removed, transcription may occur.</FONT></P>

<CENTER><FONT SIZE="+1"><IMG SRC="operon.gif" WIDTH=530 HEIGHT=298 ALIGN=bottom></FONT>

<P><FONT SIZE="+1"><IMG SRC="operon_1.gif" WIDTH=581 HEIGHT=315 ALIGN=bottom></FONT></P>

<P><FONT SIZE="+1"><IMG SRC="operon_2.gif" WIDTH=549 HEIGHT=308 ALIGN=bottom></FONT></P></CENTER>

<P><FONT SIZE="+1">The structure and operation of an operon.</FONT>
Images from Purves et al., <U>Life: The Science of Biology</U>, 4th
Edition, by Sinauer Associates (<A HREF="http://www.sinauer.com">www.sinauer.com</A>)
and WH Freeman (<A HREF="http://www.whfreeman.com">www.whfreeman.com</A>),
used with permission.</P>

<P><FONT SIZE="+1">Operons are either inducible or repressible
according to the control mechanism. Seventy-five different operons
controlling 250 structural genes have been identified for <I>E.
coli</I>. Both repression and induction are examples of negative
control since the repressor proteins turn off
transcription.</FONT></P>

<CENTER><FONT SIZE="+1"><IMG SRC="repressor_1.gif" WIDTH=471 HEIGHT=260 ALIGN=bottom></FONT>

<P><FONT SIZE="+1"><IMG SRC="repressor_2.gif" WIDTH=529 HEIGHT=279 ALIGN=bottom></FONT></P>

<P><FONT SIZE="+1"><IMG SRC="induct_1.gif" WIDTH=493 HEIGHT=250 ALIGN=bottom></FONT></P>

<P><FONT SIZE="+1"><IMG SRC="induct_2.gif" WIDTH=508 HEIGHT=282 ALIGN=bottom></FONT></P>

<P><FONT SIZE="+1"><IMG SRC="gluc_1.gif" WIDTH=509 HEIGHT=222 ALIGN=bottom></FONT></P>

<P><FONT SIZE="+2"><IMG SRC="gluc_2.gif" WIDTH=466 HEIGHT=181 ALIGN=bottom></FONT></P></CENTER>

<P><FONT SIZE="+1">Functioning of several operons.</FONT> Images from
Purves et al., <U>Life: The Science of Biology</U>, 4th Edition, by
Sinauer Associates (<A HREF="http://www.sinauer.com">www.sinauer.com</A>)
and WH Freeman (<A HREF="http://www.whfreeman.com">www.whfreeman.com</A>),
used with permission.</P>

<P><A HREF="BioBookglossPQ.html#plasmids"><FONT SIZE="+1">Plasmids</FONT></A><FONT SIZE="+1">,
small DNA fragments, are known from almost all bacterial cells.
Plasmids carry between 2 and 30 genes. Some seem to have the ability
to move in and out of the bacterial chromosome. </FONT></P>

<P><FONT SIZE="+1">An episome is a plasmid incorporated in the
bacterial chromosome. Plasmids are self-replicating in a manner like
the bacterial chromosome. Many plasmids have been recognized for
<I>E. coli</I>, including the F plasmids ("sex factors") and R
plasmids (drug/antibiotic resistance). The F plasmid contains 25
genes, some of which control the production of F pili (proteins which
extend from the surface of F<SUP>+</SUP>, or male, cells to the
surface of F<SUP>-</SUP>, or female, cells).</FONT></P>

<P><FONT SIZE="+1"><IMG SRC="bactconjug_1.gif" WIDTH=449 HEIGHT=303 ALIGN=bottom></FONT></P>

<CENTER><FONT SIZE="+1"><IMG SRC="bactconjug_2.gif" WIDTH=464 HEIGHT=309 ALIGN=bottom></FONT>

<P><FONT SIZE="+1"><IMG SRC="hfr_1.gif" WIDTH=492 HEIGHT=157 ALIGN=bottom></FONT></P></CENTER>

<P><FONT SIZE="+1">Conjugation and exchange of genetic material in
bacteria.</FONT> Images from Purves et al., <U>Life: The Science of
Biology</U>, 4th Edition, by Sinauer Associates (<A HREF="http://www.sinauer.com">www.sinauer.com</A>)
and WH Freeman (<A HREF="http://www.whfreeman.com">www.whfreeman.com</A>),
used with permission.</P>

<P><FONT SIZE="+1">The R plasmid conveys drug resistance on cells
having it. As many as 10 resistance genes can be contained on a
single R plasmid. The R plasmids can be transferred to other bacteria
of the same species, to viruses, and even to bacteria of different
species. Drug (antibiotic) resistance has been found among pathogens
causing the diseases typhoid fever, gastroenteritus, plague, undulant
fever, meningitis, and gonorrhea. In addition to the more common
modes of transfer, R plasmids may be passed through the cell
membrane. The resistance genes appear to operate by either breaking
down the antibiotics or by circumventing the block the antibiotic
places on a key bacterial metabolic pathway. </FONT></P>

<H3><FONT SIZE="+1"><A NAME=Viruses></A>Viruses | </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back
to Top</FONT></A></H3>

<P><A HREF="BioBookglossV.html#virus"><FONT SIZE="+1">Viruses</FONT></A><FONT SIZE="+1">
consist of a nucleic acid (DNA or RNA) enclosed in a protein coat
(known as a </FONT><A HREF="BioBookglossC.html#capsid"><FONT SIZE="+1">capsid</FONT></A><FONT SIZE="+1">).
The capsid may be a single protein repeated over and over, as in
tobacco mosaic virus (TMV). It may also be several different
proteins, as in the T-even </FONT><A HREF="BioBookglossB.html#bacteriophages"><FONT SIZE="+1">bacteriophages</FONT></A><FONT SIZE="+1">.
</FONT></P>

<P><FONT SIZE="+1">Once inside the cell, the nucleic acid follows one
of two paths: lytic or lysogenic. </FONT></P>

<CENTER><FONT SIZE="+1"><IMG SRC="vcycle_1.gif" WIDTH=537 HEIGHT=299 ALIGN=bottom></FONT>

<P><FONT SIZE="+1"><IMG SRC="vcycle_2.gif" WIDTH=515 HEIGHT=289 ALIGN=bottom></FONT></P>

<P><FONT SIZE="+1"><IMG SRC="vcycle_3.gif" WIDTH=529 HEIGHT=279 ALIGN=bottom></FONT></P></CENTER>

<P><FONT SIZE="+1">The lytic and lysogenic phases of a viral
replication cycle</FONT>. Images from Purves et al., <U>Life: The
Science of Biology</U>, 4th Edition, by Sinauer Associates (<A HREF="http://www.sinauer.com">www.sinauer.com</A>)
and WH Freeman (<A HREF="http://www.whfreeman.com">www.whfreeman.com</A>),
used with permission.</P>

<P><A HREF="BioBookglossR.html#retroviruses"><FONT SIZE="+1">Retroviruses</FONT></A><FONT SIZE="+1">,
such as </FONT><A HREF="BioBookglossH.html#human immunodeficiency virus (H"><FONT SIZE="+1">Human
Immunodifficiency Virus (HIV)</FONT></A><FONT SIZE="+1">, also
include the enzyme </FONT><A HREF="BioBookglossR.html#reverse transcriptase"><FONT SIZE="+1">reverse
transcriptase</FONT></A><FONT SIZE="+1"> with the viral RNA. Reverse
transcriptase makes a single-stranded viral DNA copy of the
single-stranded viral RNA. The single stranded viral DNA is
subsequently turned into a double-stranded DNA. The lytic cycle
occurs when the viral DNA immediately takes over the host cell
(remember that viruses are obligate intracellular </FONT><A HREF="BioBookglossPQ.html#parasites"><FONT SIZE="+1">parasites</FONT></A><FONT SIZE="+1">)
and begins making new viruses. Eventually the new viruses cause the
rupture (or lysis) of the cell, releasing those new viruses to
continue the infection cycle. The lysogenic cycle occurs when the
viral DNA is incorporated into the host DNA as a prophage. When the
cell replicates the prophage is passed along as if it were host DNA.
Sometimes the prophage can emerge from the host chromosome and enter
the lytic cycle spontaneously once every 10,000 cell divisions.
Ultraviolet light and x-rays may also trigger emergence of the
prophage. </FONT></P>

<P><FONT SIZE="+1">Transduction is the transfer of host DNA from one
cell to another by a virus. Some bacteriophages are temperate since
they tend to go lysogenic rather than lytic. These types of viruses
are able to transduce fragments of the host DNA.</FONT></P>

<CENTER><FONT SIZE="+1"><IMG SRC="transduc_1.gif" WIDTH=529 HEIGHT=325 ALIGN=bottom></FONT>

<P><FONT SIZE="+1"><IMG SRC="transduc_2.gif" WIDTH=498 HEIGHT=327 ALIGN=bottom></FONT></P></CENTER>

<P><FONT SIZE="+1">Induction of transduction by viruses in
bacteria.</FONT> Images from Purves et al., <U>Life: The Science of
Biology</U>, 4th Edition, by Sinauer Associates (<A HREF="http://www.sinauer.com">www.sinauer.com</A>)
and WH Freeman (<A HREF="http://www.whfreeman.com">www.whfreeman.com</A>),
used with permission.</P>

<P><FONT SIZE="+1">Transposons are DNA fragments incorporated into
the chromosomal DNA. Unlike episomes and prophages, transposons
contain a gene producing an enzyme that catalyzes insertion of the
transposon at a new site. They also have repeated sequences 20-40
nucleotides in length at each end. Insertion sequences are short
(600-1500 base pairs long) simple transposons that do not carry genes
beyond those essential for insertion of the transposon into <I>E.
coli</I>. Complex transposons are much larger and carry additional
genes. Genes incorporated in a complex transposon are known as
jumping genes since they can move about on the chromosome (even from
chromosome to chromosome). Often the complex transposons are flanked
by simple transposons.</FONT></P>

<CENTER><FONT SIZE="+1"><IMG SRC="transposon.gif" WIDTH=439 HEIGHT=327 ALIGN=bottom></FONT></CENTER>

<P><FONT SIZE="+1">Transposons and their relationship to other
genes.</FONT> Image from Purves et al., <U>Life: The Science of
Biology</U>, 4th Edition, by Sinauer Associates (<A HREF="http://www.sinauer.com">www.sinauer.com</A>)
and WH Freeman (<A HREF="http://www.whfreeman.com">www.whfreeman.com</A>),
used with permission.</P>

<H3><FONT SIZE="+1"><A NAME="The eukaryotic chromosome"></A>The
Eukaryotic Chromosome | </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back
to Top</FONT></A></H3>

<P><FONT SIZE="+1">The eukaryotic chromosome consists of DNA and
proteins that appear to play a major role in regulation of eukaryote
genes. The DNA of each chromosome is a long single molecule of double
stranded DNA. Eukaryotic DNA comes in two forms. </FONT><A HREF="BioBookglossC.html#chromatin"><FONT SIZE="+1">Chromatin</FONT></A><FONT SIZE="+1">
is the uncoiled form of DNA and is over 50% protein. </FONT><A HREF="BioBookglossC.html#chromosomes"><FONT SIZE="+1">Chromosomes</FONT></A><FONT SIZE="+1">
are coiled DNA/protein that form during the early stages of cell
division. </FONT></P>

<P><FONT SIZE="+1">The proteins associated with DNA are collectively
known as </FONT><A HREF="BioBookglossH.html#histone proteins"><FONT SIZE="+1">histones</FONT></A><FONT SIZE="+1">.
They are relatively short polypeptides which are positively charged
(basic) and thus are attracted to the negatively charged (acidic)
DNA. Histones are synthesized in quantity during the S-phase of the
</FONT><A HREF="BioBookglossC.html#cell cycle"><FONT SIZE="+1">cell
cycle</FONT></A><FONT SIZE="+1">. One function of theses proteins
seems to be the folding and packaging of DNA into chromosome form:
the 2 m of DNA in a human cell are packaged into 46 chromosomes with
a combined length of 200nm (a nm remember is 10-6m). Some 90 million
molecules of histones occur in a single cell, with the majority (30
million) being H1 histones. Five types of histone are known (H1, H2A,
H2B, H3, and H4); with the exception of H1 most eukaryote histones
are very similar.</FONT></P>

<P><FONT SIZE="+1">A </FONT><A HREF="BioBookglossN.html#nucleosomes"><FONT SIZE="+1">nucleosome</FONT></A><FONT SIZE="+1">
is the fundamental packing unit of eukaryotic DNA. The core consists
of two molecules each of H2A, H2B, H3, and H4; around which the DNA
is wound twice. The H1 histone is outside the core. Between 150-200
nucleotide pairs are associated with the core and linker DNA. This
level of packing is known as "beads on a string".</FONT></P>

<P><FONT SIZE="+1">The next level are known as the 30nm strand, whose
details of organization are not yet well known. 30 nm strands are
further condensed into 300nm wide looped domains. Looped domains are
part of condensed sections of chromosomes (the chromosome being
1400nm wide at Metaphase I).</FONT></P>

<CENTER><FONT SIZE="+1"><IMG SRC="double_helix.gif" WIDTH=356 HEIGHT=264 ALIGN=bottom></FONT>

<P><FONT SIZE="+1"><IMG SRC="chromosome.gif" WIDTH=356 HEIGHT=264 ALIGN=bottom></FONT></P></CENTER>

<P>The above images are from the University of Illinois' <A HREF="http://www.life.uiuc.edu/bio100/lessons/dna_and_protein_synthesis.html"><FONT SIZE="-1">DNA
and Protein Synthesis</FONT></A> site.</P>

<H3><FONT SIZE="+1"><A NAME="Replication of the eukaryotic c"></A>Replication
of the Eukaryotic Chromosome | </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back
to Top</FONT></A></H3>

<P><FONT SIZE="+1">Nucleotide triphosphates (in the forms ATP, GTP,
CTP and TTP) are assembled according to the </FONT><A HREF="BioBookglossS.html#semiconservative replication"><FONT SIZE="+1">semi-conservative</FONT></A><FONT SIZE="+1">
model. Other details of DNA </FONT><A HREF="BioBookglossR.html#replication"><FONT SIZE="+1">replication</FONT></A><FONT SIZE="+1">
are consistent with what we know for prokaryotes. Eukaryotic DNA has
many replication forks and also bidirectional synthesis, contrasting
to the unidirectional rolling theta prokaryotic method. Eukaryotic
DNA is synthesized much slower than prokaryotic DNA, in humans 50
nucleotides per second per replication fork. After replication the
new DNA is immediately associated with histones.</FONT></P>

<CENTER><FONT SIZE="+1"><IMG SRC="diagrep_1.gif" WIDTH=531 HEIGHT=194 ALIGN=bottom><IMG SRC="diagrep_2.gif" WIDTH=531 HEIGHT=219 ALIGN=bottom><IMG SRC="diagrep_3.gif" WIDTH=514 HEIGHT=178 ALIGN=bottom></FONT></CENTER>

<P><FONT SIZE="+1">Replication fork and its growth in a eukaryote.
</FONT>Images from Purves et al., <U>Life: The Science of
Biology</U>, 4th Edition, by Sinauer Associates (<A HREF="http://www.sinauer.com">www.sinauer.com</A>)
and WH Freeman (<A HREF="http://www.whfreeman.com">www.whfreeman.com</A>),
used with permission.</P>

<H3><FONT SIZE="+1"><A NAME="Regulation of eukaryotic gene e"></A>Regulation
of Eukaryotic Gene Expression | </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back
to Top</FONT></A></H3>

<P><FONT SIZE="+1">Eukaryotic gene regulation, especially in
multicellular organisms, is complicated by the process of development
unique to multicellular organisms. Each multicellular organism begins
as a single-celled </FONT><A HREF="BioBookglossWXYZ.html#zygote"><FONT SIZE="+1">zygote</FONT></A><FONT SIZE="+1">
which divides by </FONT><A HREF="BioBookglossM.html#mitosis"><FONT SIZE="+1">mitosis</FONT></A><FONT SIZE="+1">.
Cells differentiate into functional types by using some genes but
ignoring others. </FONT><A HREF="BioBookglossH.html#homeobox genes"><FONT SIZE="+1">Homeobox
genes</FONT></A><FONT SIZE="+1"><B> </B>establish the body plan and
position of organs in response to gradients of regulatory
molecules.<B> </B>The timing of certain gene expressions seems to
follow a sequence, such as the production of different types of fetal
hemoglobins by mammalian red blood cells, which switch to adult
hemoglobin sometime after birth. Clearly the inactivation of certain
genes occurs in every adult cell; therein lies the cure for cancer,
old age, etc.</FONT></P>

<H3><FONT SIZE="+1"><A NAME="Types of chromatin"></A>Types of
Chromatin | </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back
to Top</FONT></A></H3>

<P><FONT SIZE="+1">Heterochromatin stains more strongly and is a more
condensed chromatin. Euchromatin stains weakly and is more open (less
condensed). Euchromatin remains dispersed (uncondensed) during
Interphase, when RNA </FONT><A HREF="BioBookglossT.html#transcription"><FONT SIZE="+1">transcription</FONT></A><FONT SIZE="+1">
occurs. Some regions of heterochromatin appear to be structural (as
in the heterochromatin near the centromere region). Barr bodies,
irreversibly inactivated X-chromosomes, are also condensed
heterochromatin. Other heterochromatin regions vary from cell to
cell. As the cell differentiates, the proportion of heterochromatin
to euchromatin increases, reflecting increased specialization of the
cell as it matures. Loops (or puffs) in insect chromosomes are areas
of active RNA synthesis, suggesting again the functional genes are
located in open areas of the chromatin (or, the euchromatin).
Eukaryotes also have specific binding proteins working in a similar
fashion to prokaryotic mechanisms, however the eukaryotes, as one
would expect, have a much more complicated process.</FONT></P>

<H3><FONT SIZE="+1"><A NAME="The eukaryotic genome"></A>The
Eukaryotic Genome | </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back
to Top</FONT></A></H3>

<P><FONT SIZE="+1">We use the term </FONT><A HREF="BioBookglossG.html#genome"><FONT SIZE="+1">genome</FONT></A><FONT SIZE="+1">
to refer to all of the </FONT><A HREF="BioBookglossA.html#alleles"><FONT SIZE="+1">alleles</FONT></A><FONT SIZE="+1">
possessed by an organism (or by a population, species, or larger
taxonomic group). While the amount of DNA for a diploid cell is
constant within a species, the differences can be great between
species. Humans have 3.5 X 10<SUP>9</SUP> base pairs,
<I>Drosophila</I> has 1.5 X 10<SUP>8</SUP>, toads have 3.32 X
10<SUP>9</SUP>, and salamanders have 8 X 10<SUP>10</SUP> base pairs
per haploid genome. Much of the DNA in each cell either has no
function or has a function not yet known. Eukaryotes have only 10% of
their DNA coding for proteins. Humans may have a little as 1% coding
for proteins. Viruses and prokaryotes use a great deal more of their
DNA.</FONT></P>

<P><FONT SIZE="+1">Almost half the DNA in eukaryotic cells is
repeated </FONT><A HREF="BioBookglossN.html#nucleotide sequences"><FONT SIZE="+1">nucleotide
sequences</FONT></A><FONT SIZE="+1">. Protein-coding sequences are
interrupted by non-coding regions. Non-coding interruptions are known
as intervening sequences or </FONT><A HREF="BioBookglossI.html#intron "><FONT SIZE="+1">introns</FONT></A><FONT SIZE="+1">.
Coding sequences that are expressed are </FONT><A HREF="BioBookglossE.html#exon"><FONT SIZE="+1">exons</FONT></A><FONT SIZE="+1">.
</FONT></P>

<CENTER><FONT SIZE="+1"><IMG SRC="exintr_1.gif" WIDTH=521 HEIGHT=317 ALIGN=bottom></FONT>

<P><FONT SIZE="+1"><IMG SRC="exintrons.gif" WIDTH=531 HEIGHT=327 ALIGN=bottom></FONT></P></CENTER>

<P>Introns and exons. Images from Purves et al., <U>Life: The Science
of Biology</U>, 4th Edition, by Sinauer Associates (<A HREF="http://www.sinauer.com">www.sinauer.com</A>)
and WH Freeman (<A HREF="http://www.whfreeman.com">www.whfreeman.com</A>),
used with permission.</P>

<P><FONT SIZE="+1">Most, but not all structural eukaryote genes
contain introns. Although transcribed, these introns are excised (cut
out) before translation (a seemingly energy inefficient process). The
number of introns varies with the particular gene, even occurring in
tRNAs, rRNAs and viral genes! Generally the more complex and recently
evolved the organism, the more numerous and larger the introns. Which
came first: continuous genes lacking introns or interrupted genes
containing introns? Introns have been hypothesized to promote genetic
recombination (via </FONT><A HREF="BioBookglossC.html#crossing-over"><FONT SIZE="+1">crossing-over</FONT></A><FONT SIZE="+1">),
thus speeding up the evolution of new proteins. Exons are also
thought to code for different functional regions of
proteins.</FONT></P>

<CENTER><FONT SIZE="+1"><IMG SRC="regenk4.gif" WIDTH=567 HEIGHT=356 ALIGN=bottom></FONT>

<P><FONT SIZE="+1"><IMG SRC="regenk5.gif" WIDTH=580 HEIGHT=359 ALIGN=bottom></FONT></P></CENTER>

<P>The above images are from McGill University's Genetics Pages
(<A HREF="http://www.mcgill.ca/nrs/regenk_i.htm">http://www.mcgill.ca/nrs/regenk_i.htm</A>).</P>

<P><FONT SIZE="+1">Gene families are made up of similar, but not
identical, genes. The globin family is the best studied gene family.
</FONT><A HREF="BioBookglossH.html#hemoglobin"><FONT SIZE="+1">Hemoglobin</FONT></A><FONT SIZE="+1">
consists, in humans, of 2 </FONT><FONT SIZE="+1" FACE="Symbol">a</FONT><FONT SIZE="+1">-chains
and 2 </FONT><FONT SIZE="+1" FACE="Symbol">b</FONT><FONT SIZE="+1">-chains
clustered about a common heme. Human beta-globin genes are scattered
at five loci on human chromosome 11. These genes are expressed
sequentially during development, and are similar with same-length
introns in similar positions in each gene. Some of the genes are
inactivated copies, others are functional only during certain phases
of development. The </FONT><A HREF="BioBookglossA.html#actin"><FONT SIZE="+1">actin</FONT></A><FONT SIZE="+1">
family of genes also exhibits a similar pattern.</FONT></P>

<H3><FONT SIZE="+1"><A NAME="Transcription and processing of"></A>Transcription
and Processing of mRNA | </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back
to Top</FONT></A></H3>

<P><FONT SIZE="+1">The process of </FONT><A HREF="BioBookglossT.html#transcription"><FONT SIZE="+1">transcription</FONT></A><FONT SIZE="+1">
in eukaryotes is similar to that in prokaryotes, although there are
some differences. Eukaryote genes are not grouped in operons as are
prokaryote genes. Each eukaryote gene is transcribed separately, with
separate transcriptional controls on each gene. Whereas prokaryotes
have one type of </FONT><A HREF="BioBookglossR.html#RNA polymerase"><FONT SIZE="+1">RNA
polymerase</FONT></A><FONT SIZE="+1"> for all types of RNA,
eukaryotes have a separate RNA polymerase for each type of RNA. One
enzyme for mRNA-coding genes such as structural proteins. One enzyme
for large rRNAs. A third enzyme for smaller rRNAs and
tRNAs.</FONT></P>

<P><FONT SIZE="+1">Prokaryote </FONT><A HREF="BioBookglossT.html#translation"><FONT SIZE="+1">translation</FONT></A><FONT SIZE="+1">
begins even before transcription has finished, while eukaryotes have
the two processes separated in time and location (remember the
nuclear envelope). After eukaryotes transcribe an RNA, the
</FONT><A HREF="BioBookglossR.html#RNA transcript"><FONT SIZE="+1">RNA
transcript</FONT></A><FONT SIZE="+1"> is extensively modified before
export to the cytoplasm. A cap of 7-methylguanine (a series of an
unusual base) is added to the 5' end of the mRNA; this cap is
essential for binding the mRNA to the </FONT><A HREF="BioBookglossR.html#ribosome"><FONT SIZE="+1">ribosome</FONT></A><FONT SIZE="+1">.
A string of </FONT><A HREF="BioBookglossA.html#adenine"><FONT SIZE="+1">adenines</FONT></A><FONT SIZE="+1">
(as many as 200 nucleotides known as poly-A) is added to the 3' end
of the mRNA after transcription. The function of a poly-A tail is not
known, but it can be used to capture mRNAs for study. Introns are cut
out of the message and the exons are spliced together before the mRNA
leaves the nucleus. There are several examples of identical messages
being processed by different methods, often turning introns into
exons and vice-versa. Protein molecules are attached to mRNAs that
are exported, forming ribonucleoprotein particles (mRNPs) which may
help in transport through the </FONT><A HREF="BioBookglossN.html#nuclear pores"><FONT SIZE="+1">nuclear
pores</FONT></A><FONT SIZE="+1"> and also in attaching to ribosomes.
</FONT></P>

<H3><FONT SIZE="+1"><A NAME="Antibody-coding genes"></A>Antibody-coding
Genes | </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back to
Top</FONT></A></H3>

<P><A HREF="BioBookglossA.html#antibodies"><FONT SIZE="+1">Antibodies</FONT></A><FONT SIZE="+1">
are globular complex proteins made by multicellular individuals in
response to a specific </FONT><A HREF="BioBookglossA.html#antigens"><FONT SIZE="+1">antigen</FONT></A><FONT SIZE="+1">
(a foreign substance that has labels saying "I am a stranger"). The
cells making antibodies are </FONT><A HREF="BioBookglossL.html#lymphocytes"><FONT SIZE="+1">lymphocytes</FONT></A><FONT SIZE="+1">,
better known as </FONT><A HREF="BioBookglossWXYZ.html#white blood cell"><FONT SIZE="+1">white
blood cells</FONT></A><FONT SIZE="+1">. Antibodies immobilize and
destroy their specific antigens. A mouse can make 10,000,000
antibodies, more than the total genes of the mouse would suggest.
Antibody proteins are composed of two long and two short chains. Each
species has a constant region characteristic for the species and type
of antibody. The other region is the variable in which
antigen-specific amino-acids are located. </FONT></P>

<P><FONT SIZE="+1">Susumu Tonegawa tested (and proved) an old
hypothesis that the constant and variable regions of antibodies were
coded for by different genes. Tonegawa's work further demonstrated
that gene fragments in embryos are rearranged to form the variable
(functional) genes.</FONT></P>

<H3><FONT SIZE="+1"><A NAME="Viruses and eukaryotes"></A>Viruses and
Eukaryotes | </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back
to Top</FONT></A></H3>

<P><FONT SIZE="+1">The viruses of eukaryotes are similar to
prokaryote-infecting viruses. Proviruses are viral DNA integrated
into the host cell. Some of the DNA viruses can either initiate an
infection (lytic in prokaryotes) cycle or can form proviruses. Simian
Virus 40 (SV40) causes cancer in hamsters but not in its normal
hosts. SV40 can introduce new functional genes into the host DNA, as
can a number of other viruses.</FONT></P>

<P><A HREF="BioBookglossR.html#retroviruses"><FONT SIZE="+1">Retroviruses</FONT></A><FONT SIZE="+1">
can also insert their nucleic acid into host DNA via the
</FONT><A HREF="BioBookglossR.html#reverse transcription"><FONT SIZE="+1">reverse
transcriptase</FONT></A><FONT SIZE="+1"> mode mentioned previously.
</FONT><A HREF="BioBookglossR.html#reverse transcriptase"><FONT SIZE="+1">Reverse
transcriptase</FONT></A><FONT SIZE="+1"> is carried with the RNA into
the host cell. In the process of making the cDNA strand for the viral
RNA, the enzyme also makes long terminal repeats (LTRs) sequences at
the terminal ends of the cDNA. These LTRs may also make insertion of
the viral DNA into the host DNA easier. The inserted viral DNA makes
RNA transcripts which are packaged with viral protein coats and
reverse transcriptase. </FONT></P>

<P><FONT SIZE="+1">The viral DNA may, depending on its insertion
point, cause mutations of the host DNA. Most viral DNA insertions do
not damage the host, but rather become part of the host genome and
can be passed on if they have managed to infect a germ-line cell.
From 0.5 to 1.0% of mouse DNA may be of viral origin. </FONT></P>

<H3><FONT SIZE="+1"><A NAME="Eukaryotic transposons"></A>Eukaryotic
Transposons | </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back
to Top</FONT></A></H3>

<P><FONT SIZE="+1">Eukaryotic transposons resemble prokaryotic
transposons in many features. Many eukaryotic transposons are first
copied to RNA and then back to DNA before being inserted in a new
location.</FONT></P>

<H3><FONT SIZE="+1"><A NAME="Genes, viruses and cancer"></A>Genes,
Viruses and Cancer | </FONT><A HREF="#Table of Contents"><FONT SIZE="+1">Back
to Top</FONT></A></H3>

<P><FONT SIZE="+1">Cancer is a disease in which cells escape the
restraints on normal cell growth. Cancer is an inheritable disease
(at least from cell to daughter cells). Once a cell has become
cancerous, all of its descendant cells are cancerous. Gross
chromosomal abnormalities are often visible in cancerous cells. Most
carcinogens (cancer-generating factors) are also </FONT><A HREF="BioBookglossM.html#oncogenes"><FONT SIZE="+1">mutagens</FONT></A><FONT SIZE="+1">
(mutation-generating factors). Oncogenes are genes resembling normal
genes but in which something has gone wrong, resulting in a cancer.
Fifty oncogenes have thus far (1992) been discovered.</FONT></P>

<P><FONT SIZE="+1">Viruses seem able to cause cancer in three ways.
Presence of the viral DNA may disrupt normal host DNA functions.
Viral proteins needed for virus replication may also affect normal
host gene regulation. Since most cancer-causing viruses are
retroviruses, the virus may serve as a vector for oncogene insertion.
Transfers of genes between eukaryotic cells will allow doctors, who
have historically been limited to phenotypic cures, to attack disease
at the genotypic level. SV40 virus has been used to inject the rabbit
beta-globin gene into monkeys. Viruses can thus serve as a possible
vector to place healthy (non-mutated) alleles into eggs.</FONT></P>

<H2><A NAME=Links></A>Links | <A HREF="#Table of Contents">Back to
Top</A></H2>

<UL>
   <LI><A HREF="http://www.ai.sri.com/ecocyc/ecocyc.html">EcoCyc:
   Encyclopedia of <I>E. coli</I> Genes and Metabolism</A></LI>
   
   <LI><A HREF="http://cancer.med.upenn.edu/">OncoLink</A> The
   University of Pennsylvania Cancer Center Resource</LI>
   
   <LI><A HREF="http://www.maui.net/~southsky/introto.html">An
   Introduction to Skin Cancer</A></LI>
   
   <LI><A HREF="http://www.miaims.missouri.edu/shrp/docs/vhc.html">Breast
   Cancer Case Study</A> (the Virtual Health Care Team)</LI>
</UL>

<P>

<HR>

All text contents &copy;1992, 1994, 1997, 1999, 2000, 2001, 2007, by
M.J. Farabee. Use for educational purposes is encouraged.</P>

<P><A HREF="BioBookTOC.html"><B>Back to Table of Contents</B></A></P>

<P><STRONG>Email</STRONG>: <A HREF="mailto:mj.farabee@emcmail.maricopa.edu">mj.farabee@emcmail.maricopa.edu<IMG SRC="../envIcon.gif" WIDTH=41 HEIGHT=41 ALIGN=middle></A></P>

<P><B>Last modified: <!--#echo var="last_modified"--></B></P>

<P><B>The URL of this page is: <!--#echo var="SERVER_NAME"--><!--#echo var="SCRIPT_NAME"--></B></P>
</BODY>
</HTML>
